index.html

```diff
@@ -5,51 +5,219 @@
+                    </p>
+                    <div class="project1-button">
+                        <button type="button">html</button>
+                        <button type="button">css</button>
+                        <button type="button">JavaScript</button>
+                    </div>
+                    <div class="see-project">
+                        <button type="button">See project</button>
+                    </div>
                 </div>
-                <div class="contact">
-                    <h3>LET'S CONNECT</h3>
+            </div>
+            <div class="project" id="p3">
+                <div class="project3">
+                    <img class="profile_snapshoot" src="./image/SnapshootPortfolio3.png" alt="Profile Snapshoot">
+                    <h1 class="project1-h1">Multi-Post Stories</h1> 
+                    <ul class="project1-ul">
+                        <li id="canopy2">CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2022</li>
+                    </ul>
+                    <p class="project1-para">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="project1-button">
+                        <button type="button">html</button>
+                        <button type="button">css</button>
+                        <button type="button">JavaScript</button>
+                    </div>
+                    <div class="see-project">
+                        <button type="button">See project</button>
+                    </div>
                 </div>
-                <div class="connect-logo">
-                    <section class="logo">
-                      <ul class="logo-div">
-                        <li><a href=""><img src="./image/twiter.svg" alt=""></a></li>
-                        <li><a href=""></a><img src="./image/M.svg" alt=""></li>
-                        <li><a href=""></a><img src="./image/Vector(3).svg" alt=""></li>
-                        <li><a href=""></a><img src="./image/Vector.png" alt=""></li>
-                        <li><a href=""><img src="./image/Vector(4).svg" alt=""></a></li>
-                      </ul>
-                    </section>
+            </div>
+            <div class="project" id="p4">
+                <div class="project4">
+                    <img id="profile_snapshoot" src="./image/SnapshootPortfolio4.png" alt="Profile Snapshoot">
+                    <h1 class="project1-h1">Multi-Post Stories</h1> 
+                    <ul class="project1-ul">
+                        <li id="canopy4">CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2022</li>
+                    </ul>
+                    <p class="project1-para">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
```

Comment:
- [ ] Kindly adjust the paragraph content for the second to fourth card to start at a new line like in the first card, currently looks clustered, see screenshot below 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 1-Mobile-Main](https://user-images.githubusercontent.com/69638013/177292673-0959b563-574f-485d-ab64-4c795fca7a9a.png) |  ![Screenshot_311](https://user-images.githubusercontent.com/69638013/177292459-cc20ef70-921a-422f-b835-2c7ae17fb421.png)

Reply:
Adjusted the same.